components/script/webdriver_handlers.rs

```diff
@@ -356,6 +356,35 @@ pub fn handle_add_cookie(
         .unwrap();
 }
 
+pub fn handle_delete_cookies(
+    documents: &Documents,
+    pipeline: PipelineId,
+    reply: IpcSender<Result<(), WebDriverCookieError>>,
```

Comment:
According to the spec, the only error this should return is https://w3c.github.io/webdriver/#dfn-no-such-window. I think we can just use () instead of WebDriverCookieError and return the appropriate code from the webdriver server.